internal/controller/predicates/alloy_rules_changed_predicates.go

```diff
@@ -0,0 +1,57 @@
+package predicates
+
+import (
+	"reflect"
+	"strings"
+
+	appv1alpha1 "github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+// AlloyRulesAppChangedPredicate implements a default update predicate function on app config changes.
+type AlloyRulesAppChangedPredicate struct {
+	predicate.Funcs
+}
+
+func (AlloyRulesAppChangedPredicate) Delete(e event.DeleteEvent) bool {
+	if e.Object == nil {
+		return false
+	}
+
+	if !strings.Contains(e.Object.GetName(), "alloy-rules") {
```

Comment:
Please use the `alloyRulesAppName` variable here.

Reply:
That means I have to make it a public const but sure